rebar.config

```diff
@@ -1,7 +1,8 @@
 {erl_opts, [debug_info]}.
 {deps, [ {jsx, "2.11.0"},
-         {jose, "1.10.1"},
-         {hackney, "1.16.0"}
+         {jose, "1.11.0"},
+         {hackney, "1.16.0"},
+         {base64url, "1.0.1"}
```

Comment:
base64url used to be a dependency of 'jose' and previously we were "piggybacking" on that one. However, it is no longer a dependency of `jose` and therefore we need to include it here for `id_token` to work (which we should have done previously as well but forgot).

Reply:
Just get rid of the dependency and use what `id_token` is now using, which is `jose_base64url` module.

But actually, the dependency will stay there because it's still on the `rebar.config` of the master branch of id_token. I'll make a PR to upstream.

Reply:
> Just get rid of the dependency and use what `id_token` is now using, which is `jose_base64url` module.
> 
I am a little confused, when you say `id_token` do you mean `jose`?

>
> But actually, the dependency will stay there because it's still on the `rebar.config` of the master branch of id_token. I'll make a PR to upstream.

Do you mean this rebar.config: https://github.com/kivra/erlang-jose/blob/master/rebar.config#L8?

Reply:
I meant `erlang-jose` yes.

Yes, that `rebar.config`.

Reply:
Fixed in ea9f7df. Thanks for the feedback 👍